contents/docs/deployment/deploy-digital-ocean.md

```diff
@@ -9,12 +9,66 @@ showTitle: true
 [Digital Ocean](https://digitalocean.com) is one of the most well-established Cloud Providers. Compared to AWS, where the amount of options and configuration can be overwhelming, Digital Ocean is generally simpler to use and faster to get running. 
 <br />
 
-### Deploying to Digital Ocean
+## Deploying Posthog backed by Clickhouse
 
-To deploy on Digital Ocean, our suggested method is deploying with Docker, which we'll explain in detail on this page. If you would rather not use Docker, you can consider [deploying from source](/docs/deployment/deploy-source), as well as other providers with one-click installs, like [Heroku](/docs/deployment/deploy-heroku) or [AWS](/docs/deployment/deploy-aws).
+To deploy on Digital Ocean, our suggested method is using kubernetes to deploy Posthog backed by Clickhouse. If you would rather not use k8s or not want Clickhouse, you can consider using [docker for Postgres backed Posthog deployment](/docs/deployment/deploy-digital-ocean#deploying-posthog-backed-by-postgres), [deploying from source](/docs/deployment/deploy-source), as well as other providers with one-click installs, like [Heroku](/docs/deployment/deploy-heroku) or [AWS](/docs/deployment/deploy-aws). 
```

Comment:
s/Digital Ocean/DigitalOcean/g
(they are super strict about this as they mention it a ton in the partnership documents)

Reply:
s/Posthog/PostHog/g for obvious reasons :P

Reply:
We need to remove all options for Postgres installs, especially for new installs like this, wherever we can. We don't want anyone new using Postgres.